### PR TITLE
CSS Dashboard Update 5

### PR DESCRIPTION
**Pull request by Emily-ejag**
preview

### CODE REVIEW

File: src/components/CardAdministration.vue [hunk 1]
```diff
@@ -250,9 +248,9 @@ const toggleTable = () => {
   showTable.value = !showTable.value;
 };
 
-const isWideScreen = computed(() => {
-  return window.innerWidth > 768;
-});
+// const isWideScreen = computed(() => {
+//   return window.innerWidth > 768;
+// });
```
Comment (kellyel) [inline src/components/CardAdministration.vue:253]: If this computed function is no longer needed, we can delete
```suggestion

```

**Review by kellyel — CHANGES_REQUESTED** [review]
It's looking good so far. I left some suggestions. Could I ask you to remove all the commented code throughout the PR if we're not using it? 

I also had a look at the PR link. There are a few things I feel should be resolved (or at least let me know the thinking behind, I might be out of the loop on these decisions!)
1. There's a panel header on the admin home page, but not on the individual document pages. Is this intended?
CSS Review 1

3. If we're getting rid of panel headers on the page, could we remove the vertical lines that come from below them? (The ones that wrap around the whole form). If not, could we have those match in color?
4. If the panel headers should be present, could we give the header text a bit of padding? Right now that text is top-left aligned. 
5. In the header, the dropdown for sorting should also get a bit of vertical padding.
6. The menu button is present for participant users. That should disappear for non-admins.
7. Also on participant, it seems the 'select assignment' dropdown is in the wrong place.
CSS Review 2

9. Are there two primary colors? On the login page, the background red is very deep whereas the red on all other pages are the new lighter color.
10. On the dropdown navigation menu, could we possibly vertical center each of the items? When you hover over them they're all top-aligned. This also leads to a few of the icons looking not in-line with the text that follows it.

Also this is not the right venue for a styling discussion, however the overlapping shadows on the tabs look a bit weird to me. I understand we are trying to make them look clickable, however is there a way to make the tabs not overlap each other?
CSS Review 3

**Comment by Emily-ejag** [comment]
I believe all the changes are done

File: src/components/CreateAdministration.vue [hunk 2]
```diff
@@ -166,6 +164,9 @@ const { data: allVariants, isLoading: isLoadingVariants } = useQuery({
   staleTime: 5 * 60 * 1000, // 5 minutes
 });
 
+// const isSuperAdmin = computed(() => Boolean(userClaims.value?.claims?.super_admin));
+// const sidebarActions = ref(getSidebarActions(isSuperAdmin.value, true));
+
```
Comment (richford) [inline src/components/CreateAdministration.vue:169]: Same here.

File: src/components/CreateAdministrator.vue [hunk 3]
```diff
@@ -61,18 +58,18 @@
...
 import _cloneDeep from 'lodash/cloneDeep';
 import _union from 'lodash/union';
 import { useAuthStore } from '@/store/auth';
-import AdministratorSidebar from '@/components/AdministratorSidebar.vue';
+// import AdministratorSidebar from '@/components/AdministratorSidebar.vue';
 import OrgPicker from '@/components/OrgPicker.vue';
-import { getSidebarActions } from '@/router/sidebarActions';
-import { fetchDocById } from '@/helpers/query/utils';
+// import { getSidebarActions } from '@/router/sidebarActions';
+// import { fetchDocById } from '@/helpers/query/utils';
```
Comment (richford) [inline src/components/CreateAdministrator.vue:72]: Same comment here about preferring deletion over commenting.

File: src/components/CreateOrgs.vue [hunk 4]
```diff
@@ -217,7 +214,7 @@ const { isLoading: isLoadingClaims, data: userClaims } = useQuery({
 
 const isSuperAdmin = computed(() => Boolean(userClaims.value?.claims?.super_admin));
 const adminOrgs = computed(() => userClaims.value?.claims?.minimalAdminOrgs);
-const sidebarActions = ref(getSidebarActions(isSuperAdmin.value, true));
+// const sidebarActions = ref(getSidebarActions(isSuperAdmin.value, true));
```
Comment (richford) [inline src/components/CreateOrgs.vue:217]: Same comment here about preferring deletion over commenting.

File: src/components/ListOrgs.vue [hunk 5]
```diff
@@ -52,20 +49,26 @@
...
+                :loading="isLoading || isLoadingCount || isFetching || isFetchingCount"
+                @page="onPage($event)"
+                @sort="onSort($event)"
+                @export-all="exportAll"
+              />
+              <AppSpinner v-else />
+          </div>
+          <div v-else>
+            <AppSpinner/>
+            <p class="text-center">Still Loading Organization data</p>
```
Comment (richford) [inline src/components/ListOrgs.vue:70]:
```suggestion
            <p class="text-center">Loading organization data</p> 
```

File: src/components/ListOrgs.vue [hunk 6]
```diff
@@ -130,7 +131,7 @@ const { isLoading: isLoadingClaims, data: userClaims } = useQuery({
 
 const isSuperAdmin = computed(() => Boolean(userClaims.value?.claims?.super_admin));
 const adminOrgs = computed(() => userClaims.value?.claims?.minimalAdminOrgs);
-const sidebarActions = ref(getSidebarActions(isSuperAdmin.value, true));
+// const sidebarActions = ref(getSidebarActions(isSuperAdmin.value, true));
```
Comment (richford) [inline src/components/ListOrgs.vue:134]: Same comment here about preferring deletion over commenting.

File: src/components/ListOrgs.vue [hunk 7]
```diff
@@ -227,6 +228,7 @@ const exportAll = async () => {
 };
 
 const tableColumns = computed(() => {
+  console.log("I am loading the name, abbreviation, address and tags")
```
Comment (richford) [inline src/components/ListOrgs.vue:231]:
```suggestion
```

File: src/components/ListOrgs.vue [hunk 8]
```diff
@@ -235,13 +237,15 @@ const tableColumns = computed(() => {
   ];
 
   if (['districts', 'schools'].includes(activeOrgType.value)) {
-    columns.push(
+    console.log("I am loading the districts and schools")
```
Comment (richford) [inline src/components/ListOrgs.vue:240]:
```suggestion
```

File: src/components/ListOrgs.vue [hunk 9]
```diff
@@ -235,13 +237,15 @@ const tableColumns = computed(() => {
...
+    columns.push(      
       { field: 'mdrNumber', header: 'MDR Number', dataType: 'string' },
       { field: 'ncesId', header: 'NCES ID', dataType: 'string' },
     );
   }
 
   if (['districts', 'schools', 'classes'].includes(activeOrgType.value)) {
+    console.log("I am loading the districts, schools and classess")
```
Comment (richford) [inline src/components/ListOrgs.vue:248]:
```suggestion
```

File: src/components/ListUsers.vue [hunk 10]
```diff
@@ -26,34 +23,20 @@
...
-import { getSidebarActions } from '../router/sidebarActions';
+import { ref, onMounted } from 'vue';
 import { useAuthStore } from '@/store/auth';
 import _isEmpty from 'lodash/isEmpty';
 import { useQuery } from '@tanstack/vue-query';
 import AppSpinner from './AppSpinner.vue';
 import { storeToRefs } from 'pinia';
 import { countUsersByOrg, fetchUsersByOrg } from '@/helpers/query/users';
-import { fetchDocById } from '../helpers/query/utils';
+// import { fetchDocById } from '../helpers/query/utils';
```
Comment (richford) [inline src/components/ListUsers.vue:33]: Same comment here about preferring deletion over commenting.

File: src/components/NavBar.vue [hunk 11]
```diff
@@ -18,15 +27,38 @@
 </template>
 
 <script setup>
-import { ref } from 'vue';
+import { ref,computed } from 'vue';
```
Comment (richford) [inline src/components/NavBar.vue:30]:
```suggestion
import { ref, computed } from 'vue';
```

File: src/components/NavBar.vue [hunk 12]
```diff
@@ -18,15 +27,38 @@
...
 import { useRouter } from 'vue-router';
 import { storeToRefs } from 'pinia';
 import { useAuthStore } from '@/store/auth';
 import _get from 'lodash/get';
+// import AdministratorSidebar from './AdministratorSidebar.vue';
+ import {getSidebarActions} from '@/router/sidebarActions'
+ import { fetchDocById } from '@/helpers/query/utils';
+ import { useQuery } from '@tanstack/vue-query';
```
Comment (richford) [inline src/components/NavBar.vue:38]:
```suggestion
 import { getSidebarActions } from '@/router/sidebarActions'
 import { fetchDocById } from '@/helpers/query/utils';
 import { useQuery } from '@tanstack/vue-query';
```

File: src/components/NavBar.vue [hunk 13]
```diff
@@ -18,15 +27,38 @@
...
+
+ const isSuperAdmin = computed(() => Boolean(userClaims.value?.claims?.super_admin));
+// const sidebarActions = ref(getSidebarActions(isSuperAdmin.value, true));
+
+
+
+// const menuItems = [
+
+//   `<li>${}</li>`
+// ]
```
Comment (richford) [inline src/components/NavBar.vue:61]:
```suggestion
```

File: src/components/NavBar.vue [hunk 14]
```diff
@@ -41,7 +73,29 @@ const { roarfirekit } = storeToRefs(authStore);
...
 
-// const menu = ref();
+const isAtHome = computed(()=>{
+  return router.currentRoute.value.fullPath === '/'
+});
+
+// const getActions = computed(()=>{
+//   return getSidebarActions(isSuperAdmin.value, !isAtHome.value)
+// })
+// const sidebarActions = ref(getSidebarActions(authStore.isUserSuperAdmin, !isAtHome.value));
```
Comment (richford) [inline src/components/NavBar.vue:83]:
```suggestion
```

File: src/components/RoarDataTable.vue [hunk 15]
```diff
@@ -75,8 +73,10 @@
       @select-all-change="onSelectAll"
       @row-select="onSelectionChange"
       @row-unselect="onSelectionChange"
+      
+      
     >
-      <PvColumn selection-mode="multiple" header-style="width: 3rem" :reorderable-column="false" frozen />
+      <PvColumn selection-mode="multiple" header-style="width:3rem; " :reorderable-column="false" frozen />
```
Comment (richford) [inline src/components/RoarDataTable.vue:79]:
```suggestion
      <PvColumn selection-mode="multiple" header-style="width: 3rem;" :reorderable-column="false" frozen />
```

File: src/components/RoarDataTable.vue [hunk 16]
```diff
@@ -111,7 +112,7 @@
               :severity="_get(colData, col.severityField)"
               :value="_get(colData, col.field)"
               :icon="_get(colData, col.iconField)"
-              :style="`background-color: ${_get(colData, col.tagColor)}; min-width: 2rem; ${
+              :style="`background-color: ${_get(colData, col.tagColor)}; min-width: 2rem; color:white; ${
```
Comment (richford) [inline src/components/RoarDataTable.vue:115]:
```suggestion
              :style="`background-color: ${_get(colData, col.tagColor)}; min-width: 2rem; color: white; ${
```

File: src/components/RoarDataTable.vue [hunk 17]
```diff
@@ -217,10 +220,13 @@ import _get from 'lodash/get';
 import _set from 'lodash/set';
 import _map from 'lodash/map';
 import _forEach from 'lodash/forEach';
+// import _debounce from 'lodash/debounce';
```
Comment (richford) [inline src/components/RoarDataTable.vue:223]:
```suggestion
```

File: src/components/RoarDataTable.vue [hunk 18]
```diff
@@ -217,10 +220,13 @@ import _get from 'lodash/get';
...
 import _find from 'lodash/find';
 import _filter from 'lodash/filter';
 import _toUpper from 'lodash/toUpper';
 import _startCase from 'lodash/startCase';
+import _without from 'lodash/without';
+// import PvInputSwitch from 'primevue/inputswitch';
```
Comment (richford) [inline src/components/RoarDataTable.vue:229]:
```suggestion
```

File: src/components/tasks/RegisterTask.vue [hunk 19]
```diff
@@ -49,7 +49,7 @@
             </section>
             <!--Task URL-->
             <section class="form-section">
-              <div>
+              <div class="taskURLs">
```
Comment (richford) [inline src/components/tasks/RegisterTask.vue:52]: I'd like to use kebab case for css classes.

```suggestion
              <div class="task-urls">
```

File: src/components/tasks/RegisterTask.vue [hunk 20]
```diff
@@ -69,7 +69,7 @@
             </section>
             <!-- Cover Image -->
             <section class="form-section">
-              <div>
+              <div class="taskURLs">
```
Comment (richford) [inline src/components/tasks/RegisterTask.vue:72]: Same here
```suggestion
              <div class="task-urls">
```

File: src/components/tasks/RegisterTask.vue [hunk 21]
```diff
@@ -100,8 +100,9 @@
               <PvButton icon="pi pi-trash" class="p-button-danger delete-btn" @click="removeField(taskParams, index)" />
             </div>
           </div>
-
-          <PvButton label="Add Field" class="p-button-success" @click="addField(taskParams)" />
+          <div class="addField">
```
Comment (richford) [inline src/components/tasks/RegisterTask.vue:103]: Same here
```suggestion
          <div class="add-field">
```

File: src/components/tasks/RegisterTask.vue [hunk 22]
```diff
@@ -442,5 +443,30 @@ function resetVariantForm() {
 .params-container {
   display: flex;
   margin-bottom: 1rem;
+  margin-left: 10%;
+  margin-right: 10%;
+}
+.addField, .form-submit  {
```
Comment (richford) [inline src/components/tasks/RegisterTask.vue:449]:
```suggestion
.add-field, .form-submit  {
```

File: src/components/tasks/RegisterTask.vue [hunk 23]
```diff
@@ -442,5 +443,30 @@ function resetVariantForm() {
...
+.p-fluid .p-button {
+  width: 50%;
+}
+.p-fluid .p-input-icon-left, .p-fluid .p-input-icon-right {
+  display: block;
+  width: 80%;
+  margin-left: 10%;
+  margin-right: 10%;
+}
+.p-dropdown, .taskURLs {
```
Comment (richford) [inline src/components/tasks/RegisterTask.vue:464]:
```suggestion
.p-dropdown, .task-urls {
```

File: src/components/tasks/RegisterTask.vue [hunk 24]
```diff
@@ -199,8 +200,9 @@
               />
             </div>
           </div>
-
-          <PvButton label="Add Field" class="p-button-success" @click="addField(variantParams)" />
+          <div class="addField">
```
Comment (richford) [inline src/components/tasks/RegisterTask.vue:203]:
```suggestion
          <div class="add-field">
```

File: src/pages/AdministrationProgress.vue [hunk 25]
```diff
@@ -42,8 +39,8 @@ import _isEmpty from 'lodash/isEmpty';
 import _kebabCase from 'lodash/kebabCase';
 import _map from 'lodash/map';
 import { useAuthStore } from '@/store/auth';
-import AdministratorSidebar from '@/components/AdministratorSidebar.vue';
-import { getSidebarActions } from '@/router/sidebarActions';
+// import AdministratorSidebar from '@/components/AdministratorSidebar.vue';
+// import { getSidebarActions } from '@/router/sidebarActions';
```
Comment (richford) [inline src/pages/AdministrationProgress.vue:43]: Same comment here about preferring deletion over commenting.

File: src/pages/AdministrationProgress.vue [hunk 26]
```diff
@@ -53,7 +50,7 @@ import { taskDisplayNames } from '@/helpers/reports.js';
 
 const authStore = useAuthStore();
 
-const sidebarActions = ref(getSidebarActions(authStore.isUserSuperAdmin, true));
+// const sidebarActions = ref(getSidebarActions(authStore.isUserSuperAdmin, true));
```
Comment (richford) [inline src/pages/AdministrationProgress.vue:53]: Same comment here about preferring deletion over commenting.

File: src/pages/HomeAdministrator.vue [hunk 27]
```diff
@@ -69,9 +67,9 @@ import { computed, ref, onMounted } from 'vue';
 import { storeToRefs } from 'pinia';
 import { orderByDefault, fetchDocById } from '@/helpers/query/utils';
 import { administrationCounter, administrationPageFetcher } from '../helpers/query/administrations';
-import { getSidebarActions } from '../router/sidebarActions';
+// import { getSidebarActions } from '../router/sidebarActions';
```
Comment (richford) [inline src/pages/HomeAdministrator.vue:70]: Same comment here about preferring deletion over commenting.

File: src/pages/HomeAdministrator.vue [hunk 28]
```diff
@@ -69,9 +67,9 @@ import { computed, ref, onMounted } from 'vue';
...
 import CardAdministration from '@/components/CardAdministration.vue';
-import AdministratorSidebar from '@/components/AdministratorSidebar.vue';
+// import AdministratorSidebar from '@/components/AdministratorSidebar.vue';
```
Comment (richford) [inline src/pages/HomeAdministrator.vue:72]: Same comment here about preferring deletion over commenting.

File: src/pages/HomeAdministrator.vue [hunk 29]
```diff
@@ -83,12 +81,12 @@ const authStore = useAuthStore();
...
+// const sidebarActions = ref(getSidebarActions(authStore.isUserSuperAdmin, false));
 
-const userInfo = ref({
-  name: 'Admin name',
-  district: 'District Name',
-});
+// const userInfo = ref({
+//   name: 'Admin name',
+//   district: 'District Name',
+// });
```
Comment (richford) [inline src/pages/HomeAdministrator.vue:89]: Same comment here about preferring deletion over commenting.

File: src/pages/RegisterStudents.vue [hunk 30]
```diff
@@ -138,8 +134,8 @@ import _uniqBy from 'lodash/uniqBy';
 import _startCase from 'lodash/startCase';
 import { useAuthStore } from '@/store/auth';
 import { useRouter } from 'vue-router';
-import AdministratorSidebar from '@/components/AdministratorSidebar.vue';
-import { getSidebarActions } from '../router/sidebarActions';
+// import AdministratorSidebar from '@/components/AdministratorSidebar.vue';
+// import { getSidebarActions } from '../router/sidebarActions';
```
Comment (richford) [inline src/pages/RegisterStudents.vue:138]: Same comment here about preferring deletion over commenting.

File: src/pages/RegisterStudents.vue [hunk 31]
```diff
@@ -150,7 +146,7 @@ const toast = useToast();
 const isFileUploaded = ref(false);
 const rawStudentFile = ref({});
 
-const sidebarActions = ref(getSidebarActions(authStore.isUserSuperAdmin, true));
+// const sidebarActions = ref(getSidebarActions(authStore.isUserSuperAdmin, true));
```
Comment (richford) [inline src/pages/RegisterStudents.vue:149]: Same comment here about preferring deletion over commenting.

File: src/pages/ScoreReport.vue [hunk 32]
```diff
@@ -636,13 +585,36 @@ function getScoreKeys(row, grade) {
...
+//   } else return [];
+// });
+
+// const emptyTagColorMap = {
+//   above: 'green',
+//   some: '#edc037',
+//   below: '#c93d82',
+// };
+
+
```
Comment (richford) [inline src/pages/ScoreReport.vue:610]: Same comment here about preferring deletion over commenting.

File: src/pages/ScoreReport.vue [hunk 33]
```diff
@@ -105,7 +95,16 @@
             @filter="onFilter($event)"
             @export-all="exportAll"
             @export-selected="exportSelected"
-          />
+          >
+          <label for="ms-columns" class="view-label">View</label>
+          <PvDropdown
+            id="view-columns"
```
Comment (richford) [inline src/pages/ScoreReport.vue:101]: Shouldn't this `id` be an `input-id` and should it match the `for` param of the `label` element?
```suggestion
          <label for="view-columns" class="view-label">View</label>
          <PvDropdown
            input-id="view-columns"
```

File: src/components/tasks/RegisterTask.vue (hunk 20 above)
Comment (Emily-ejag) [reply src/components/tasks/RegisterTask.vue]: done

File: src/components/tasks/RegisterTask.vue (hunk 21 above)
Comment (Emily-ejag) [reply src/components/tasks/RegisterTask.vue:103]: done

**Comment by richford** [comment]
Hi @Emily-ejag,

I've been thinking about this PR and have decided that it would be best to break it up into multiple PRs. There are some of the style changes that I'm not quite ready to sign off on yet. I'd still like to workshop some things. But then there are other improvements that are ready to go (e.g., moving the menu bar, adding a compact table view). I think it would be easier to merge multiple small PRs with logically distinct changes.

I really appreciate all of the hard work you've done on this PR. And I know it's frustrating to have me come back after multiple reviews and merge conflicts only to ask you to resubmit as multiple smaller PRs, but I think it'll be worth the effort. Can you close this without merging and then resubmit multiple PRs as follows:
1. Moving the menu button to the nav bar (using `PvMenu` instead of `Menu` to maintain consistency with the rest of the site
2. Adding compact table views
3. Modifying panels and headers
4. Changing colors
5. Anything else that I missed.

**Comment by richford** [comment]
If it helps, you can use `git cherry-pick` to easily break up this branch into multiple smaller branches.

**Pull request closed without merging**